netbox/extras/models/configs.py

```diff
@@ -182,7 +182,7 @@ def get_config_context(self):
 
         if not hasattr(self, 'config_context_data'):
             # The annotation is not available, so we fall back to manually querying for the config context objects
-            config_context_data = ConfigContext.objects.get_for_object(self, aggregate_data=True)
+            config_context_data = ConfigContext.objects.get_for_object(self, aggregate_data=True) or []
```

Comment:
Looks like a potential bug fix? Maybe worth capturing in a separate issue.

Reply:
With Django 5 the tests won't pass without this change so included it here.  I wasn't completely sure what change caused this to fail, but it looked like a valid fix so didn't dig into what Django 5 issue caused it.